lib/key-info.js

```diff
@@ -0,0 +1,55 @@
+const xpath = require("xpath");
+
+/**
+ * A basic, string-based, implementation of a KeyInfoProvider
+ *
+ * @param {string} cert the public certificate
+ * @implements {KeyInfoProvider}
+ */
+class KeyInfo {
+  constructor(cert) {
+    this.cert = cert;
+  }
+  /**
+   * Builds the contents of a KeyInfo element as an XML string.
+   *
+   * For example, if the value of the prefix argument is 'foo', then
+   * the resultant XML string will be "<foo:X509Data></foo:X509Data>"
+   *
+   * @param cert the public key as a string or array of strings (see https://www.w3.org/TR/2008/REC-xmldsig-core-20080610/#sec-X509Data)
+   * @param prefix an optional namespace alias to be used for the generated XML
+   * @return an XML string representation of the contents of a KeyInfo element
+   */
+  makeKeyInfo(cert = "", prefix = "") {
```

Comment:
I am not a fan of this rename. If your concern is the overlap with `SignedXml.getKeyInfo` method, maybe you should consider renaming the other one.